tests/phpunit/unit-tests/api/utilities/beansSanitizePath.php

```diff
@@ -29,21 +29,102 @@ protected function setUp() {
 		require_once BEANS_TESTS_LIB_DIR . 'api/utilities/functions.php';
 	}
 
+	/**
+	 * Test beans_sanitize_path() should remove Windows drive.
+	 */
+	public function test_should_remove_windows_drive() {
+		$this->assertSame( '/Program Files/tmp/', beans_sanitize_path( 'C:\Program Files\tmp\\' ) );
+		$this->assertSame( '/Foo/bar/baz/index.txt', beans_sanitize_path( 'D:\Foo\bar\baz\index.txt' ) );
+	}
+
```

Comment:
Check first that it removes the Window's drive from the path before doing the rest of the tests.